python/ray/tests/test_runtime_env_plugin.py

```diff
@@ -181,27 +188,21 @@ def validate(runtime_env_dict: dict) -> str:
 class HangPlugin(DummyPlugin):
     name = HANG_PLUGIN_NAME
 
-    def create(
-        self, uri: str, runtime_env: "RuntimeEnv", ctx: RuntimeEnvContext  # noqa: F821
-    ) -> float:
-        sleep(3600)
-
-
-class DiasbleTimeoutPlugin(DummyPlugin):
```

Comment:
Why we delete this plugin?

Reply:
It looks like it was never used for anything!